src/com/amazon/ion/benchmark/OptionsCombinationBase.java

```diff
@@ -51,6 +52,7 @@ abstract class OptionsCombinationBase {
     final int limit;
     final boolean jsonUseBigDecimals;
     final boolean autoFlush;
+    final Integer ionMinorVersion;
```

Comment:
I wish we could just use `ionVersion` and accept a string such as "1.1" or "1.0", but I get that this is much simpler.